definitions/infra-azureapimanagementservice/definition.yml

```diff
@@ -8,3 +8,11 @@ goldenTags:
 configuration:
   entityExpirationTime: DAILY
   alertable: true
+synthesis:
+  rules:
+  - identifier: azure.resourceId
```

Comment:
Just for my knowledge. what kind of values we get here? similar to aws arn's?

Reply:
It-s unique like an ARN, but it's built more like the path in a file system, with slashes. The structure is something like:
```
/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.{serviceName}/{resourceType}/{resourceName}
```

form that, the substring `Microsoft.{serviceName}/{resourceType}` would be your `resourceType`.